tests/fluka/test_fluka_beam.py

```diff
@@ -20,16 +20,49 @@ def test_parse_fluka_beam(project_fluka_json):
     assert isclose(fluka_beam.beam_dir[1], 0.0, abs_tol=1e-16)
 
 
-def test_parse_heavy_ions(project_fluka_json):
+def test_parse_fluka_beam_MeV_nucl(project_fluka_json):
     """Test if Fluka beam is parsed correctly"""
     beam_json = copy.deepcopy(project_fluka_json['beam'])
+    beam_json['energyUnit'] = 'MeV/nucl'
+    beam_json['energy'] = 100
+    beam_json['particle']['id'] = 23
+    beam_json['particle']['a'] = 3
+
+    fluka_beam = parse_beam(beam_json)
+
+    assert fluka_beam.energy_MeV == 300
+    assert fluka_beam.particle_name == '3-HELIUM'
+
+
+def test_parse_heavy_ions_MeV(project_fluka_json):
+    """Test if Fluka beam is parsed correctly"""
+    beam_json = copy.deepcopy(project_fluka_json['beam'])
+    beam_json['energy'] = 300
+    beam_json['energyUnit'] = 'MeV'
     beam_json['particle']['id'] = 25
     beam_json['particle']['a'] = 6
     beam_json['particle']['z'] = 12
 
     fluka_beam = parse_beam(beam_json)
 
-    assert fluka_beam.energy_MeV == 70
+    assert fluka_beam.energy_MeV == 50
+    assert fluka_beam.particle_name == 'HEAVYION'
+    assert fluka_beam.heavy_ion_a == 6
+    assert fluka_beam.heavy_ion_z == 12
```

Comment:
Please cross check this file:
- Z is atomic number, its number of protons in nuclei https://en.wikipedia.org/wiki/Atomic_number
- A is mass number, its total number of protons and neutrons in nuclei https://en.wikipedia.org/wiki/Mass_number

By definition A is greater of equal than Z.
Here we have A=6 and Z=12.

For carbon ion that should be opposite.

I know that the arithmetic works for this case as well, but its very confusing and may be source of future errors.